tests/introspection/tests.py

```diff
@@ -189,6 +184,15 @@ def test_get_indexes_multicol(self):
         self.assertNotIn('first_name', indexes)
         self.assertIn('id', indexes)
 
+    @skipUnlessDBFeature('can_introspect_index_type')
```

Comment:
Should I keep this flag on this test in favour of 3rd party backends?
If no, there's no reason to keep this flag at all.